vignettes/Model.Rmd

```diff
@@ -10,90 +12,365 @@ vignette: >
+
+Griffin, J. T., Déirdre Hollingsworth, T., Reyburn, H., Drakeley, C. J., Riley, E. M., & Ghani, A. C. (2015). Gradual acquisition of immunity to severe malaria with increasing exposure. Proceedings of the Royal Society B: Biological Sciences, 282(1801). <https://doi.org/10.1098/rspb.2014.2657>
+
+Griffin, J. T., Bhatt, S., Sinka, M. E., Gething, P. W., Lynch, M., Patouillard, E., Shutes, E., Newman, R. D., Alonso, P., Cibulskis, R. E., & Ghani, A. C. (2016). Potential for reduction of burden and local elimination of malaria by reducing Plasmodium falciparum malaria transmission: A mathematical modelling study. The Lancet Infectious Diseases, 16(4), 465--472. [https://doi.org/10.1016/S1473-3099(15)00423-5](https://doi.org/10.1016/S1473-3099(15)00423-5)
+
+# Basic simulation
+
+### Simulation code
+
+The key package function is `run_simulation()` which, in its most basic form, simply requires a number of timesteps in days. Default parameter settings assume a human population of size of 100, an initial mosquito population size of 1000 with all species in equal proportions, with no treatment interventions and no seasonality. The full parameters list can be seen in the documentation for `get_parameters()`.
+
+```{r, output.lines=6}
+library(malariasimulation)
+test_sim <- run_simulation(timesteps = 100)
+```
+
+### Output
+
+The `run_simulation()` function then simulates malaria transmission dynamics and returns a dataframe for the following parameters through time:
+
+-   `infectivity`: human infectiousness
+-   `EIR_All`: the entomological inoculation rate
+-   `FOIM`: the force of mosquito infection
+-   `mu_All`: adult mosquito death rate
+-   `n_bitten`: the number of infectious bites
+-   `n_infections`: the number human infections
+-   `natural_deaths`: deaths from old age
+-   `S_count`, `A_count`, `D_count`, `U_count`, `Tr_count`: the human population size in each state
+-   `ica_mean`: mean acquired immunity to clinical infection
+-   `icm_mean`: mean maternal immunity to clinical infection
+-   `ib_mean`: mean blood immunity to all infection
+-   `id_mean`: mean immunity from detected using microscopy
+-   `iva_mean`: mean acquired immunity to severe infection
+-   `ivm_mean`: mean maternal immunity to severe infection
+-   `n_730_3650`: population size of an age group of interest (where the default is set to 730-3650 days old, or 2-10 years, but which may be adjusted (see [Demography](https://mrc-ide.github.io/malariasimulation/articles/Demography.html) vignette for more details)
+-   `n_detect_730_3650`: number with possible detection through microscopy of a given age group
+-   `p_detect_730_3650`: the sum of probabilities of detection through microscopy of a given age group
+-   `E_All_count`, `L_All_count`, `P_All_count`, `Sm_All_count`, `Pm_All_count`, `Im_All_count`, `NonExistent_All_count`: mosquito population sizes in each state
```

Comment:
Check the nonexistent var - I can't get it to work on mine!

Reply:
Hmmm not sure about this. It appears in mine...?
![image](https://user-images.githubusercontent.com/73181123/229089976-bb392b00-d780-4eb5-b990-3fc3448f391b.png)

Reply:
I got Tom Brewer to check this part of the code too, and he gets NonExistent_All_count as a column too.